packages/x-charts/src/internals/plugins/models/plugin.ts

```diff
@@ -13,11 +13,6 @@ export interface ChartPluginOptions<TSignature extends ChartAnyPluginSignature>
    * The parameters after being processed with the default values.
    */
   params: ChartUsedDefaultizedParams<TSignature>;
-  /**
-   * The store of controlled properties.
-   * If they are not controlled by the user, they will be initialized by the plugin.
-   */
-  models: ChartUsedControlModels<TSignature>;
```

Comment:
I suppose we don't use models?

Reply:
I'm removing the only usage which was `useChartHighlight.models`